README.md

```diff
@@ -1,2 +1,49 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or owner) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (participants) have perfomed the specified tasks and there haven't been any malicious changes in the files.
+
+in-toto requires a project layout that specifies the functionaries and the tasks they are supposed to perform.
+After each functionary performs its task, a link metadata is generated.
+This metadata is used to verify the intermediate and final products with the project layout.
+
+# in-toto demo
+You can try in-toto by running the demo application at [https://github.com/in-toto/in-toto/tree/develop/demo].
```

Comment:
Please use GitHub's markdown link markup, e.g. `[demo application](https://github.com/in-toto/in-toto/tree/develop/demo)`